xml/System.Net/IPAddress.xml

```diff
@@ -196,9 +196,22 @@
         <Parameter Name="address" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="address">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="address">The byte span value of the IP address.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.IPAddress" /> class with the address specified as a byte span.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Net.IPAddress> is created with the <xref:System.Net.IPAddress.Address> property set to `address`.  
+  
+ If the length of `address` is 4, this method constructs an IPv4 address; otherwise, an IPv6 address with a scope of 0 is constructed.  
+  
+ The byte span is assumed to be in network byte order with the most significant byte first in index position 0.  
```

Comment:
Should these be documented in the `param` rather than `remarks`?

Reply:
+1
This is an interesting suggestion. We should always keep in mind that remarks do not show up in intellisense, so if we add this information at the end of the param, we ensure developers have this information available first hand.

Reply:
That's one reason, but also these remarks are all specific to the one param.

Reply:
```suggestion
```